.travis.yml

```diff
@@ -1,21 +1,28 @@
-language: scala
-jdk: oraclejdk8
 sudo: required
 
-cache:
-  directories:
-    - $HOME/.ivy2
-    - $HOME/.sbt
-
-services:
-  - docker
+language: scala
+jdk: oraclejdk8
 
 scala:
   - 2.10.6
   - 2.11.7
 
+before_install:
+  - openssl aes-256-cbc -K $encrypted_e4898ca93742_key -iv $encrypted_e4898ca93742_iv -in secrets.tgz.enc -out secrets.tgz -d
+  - tar zxf secrets.tgz
+
 before_script:
-  - sbt update # update dependencies here in order to cache them and mount them later to docker container
+  - sudo curl -fsSL https://get.docker.com/ | sh
+  - sudo docker run --name riak-ts -d -p 8087:8087 -p 8098:8098 basho/riak-ts
```

Comment:
👍 since it looks like the build and test are totaling around 16 minutes right now so increasing to about half an hour isn't too terrible. Plus when we move to scalatest and parallelized tests, the time should shrink.